tilerator/runserver.sh

```diff
@@ -2,8 +2,8 @@
 set -e
 
 # wait for cassandra to be up
-CASSANDRA_SERVER=cassandra
-CASSANDRA_PORT=9042
+CASSANDRA_SERVER="${CASSANDRA_HOST:-cassandra}"
+CASSANDRA_PORT="${CASSANDRA_PORT:-9042}"
```

Comment:
To use a different port, this change would not be enough. The usage of the "cassandra-driver" would need to be adapted in kartotherian/tilerator implementation. Is it an immediate need?